internal/git/utils_test.go

```diff
@@ -85,19 +85,70 @@ func TestCheckDirty(t *testing.T) {
 	InitRepo(t)
 	EmptyCommit(t, "this is a test")
 
-	out, err := CheckDirty()
+	out, err := CheckDirty(nil)
 	require.NoError(t, err)
 	assert.Empty(t, out)
 }
 
+
+func TestCheckDirty_allowedFiles(t *testing.T) {
```

Comment:
Please correct the case used on these test names e.g. `TestCheckDirty_AllowedFiles`